app/views/overrides/activation_keys/_host_media_type_select.html.erb

```diff
@@ -16,19 +16,29 @@
 %>
 
 <div class="clear-fix" id="media_selection_section">
-  <div class="form-group ">
-     <label class="col-md-2 control-label" for="use_install_media"><%= _('Media Selection') %></label>
+  <div class="form-group">
+     <label class="col-md-2 control-label" for="use_install_media">
+        <%= _('Media Selection') %>
+     </label>
      <div class="col-md-4">
         <label class="radio-inline">
-          <input data-media-selector="true" id="host_use_synced_content" name="media_selector" value="synced_content" type="radio" <%= synced_content_radio  %> <%= synced_content_disabled %>/> <%= _('Synced Content') %>
+          <input data-media-selector="true" id="host_use_synced_content" name="media_selector" value="synced_content" type="radio" 
+            <%= synced_content_radio  %> 
+            <%= synced_content_disabled %>
+          /> 
+          <%= _('Synced Content') %>
+          <%= popover("", "To enable the synced content option, first select a content view and lifecycle environment containing synced kickstart repositories.") %>
```

Comment:
I verified that the Content Source also needs to be set, along with the Architecture and Operating system. For completeness, I think we should include those here as well.

Reply:
Awesome. The message should be fixed, thank you.